backend/core/followup_management_service.py

```
@@ -0,0 +1,278 @@
+                        elif action == "activate":
+                            success = self.db_manager.update_followup_question(question_id, is_active=True)
+                        elif action == "deactivate":
+                            success = self.db_manager.update_followup_question(question_id, is_active=False)
+                        elif action == "update":
+                            success = self.db_manager.update_followup_question(
+                                question_id,
+                                question_text=operation.get("question_text"),
+                                sort_order=operation.get("sort_order"),
+                            )
+                        else:
+                            raise ValueError(f"Unknown action: {action}")
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)